src/commands/games/overwatch.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  ChatInputCommandInteraction,
+  SlashCommandBuilder,
+  EmbedBuilder,
+} from 'discord.js'
+import CommandBuilder from '~/classes/command.classes'
+import { getHeroes } from '~/helpers/overwatch.helper'
+
+const data = new SlashCommandBuilder()
+  .setName('overwatch')
+  .setDescription('Picks a random hero for you')
+  .addStringOption((option) =>
+    option
+      .setName('role')
+      .setDescription('Pick a role')
+      .setRequired(true)
+      .addChoices(
+        { name: 'damage', value: 'damage' },
+        { name: 'support', value: 'support' },
+        { name: 'tank', value: 'tank' }
+      )
+  )
+
+async function execute(interaction: ChatInputCommandInteraction) {
+  const heroes = await getHeroes(interaction.options.getString('role')!)
+
+  if (typeof heroes === 'string') {
+    interaction.reply(heroes)
+    return
+  }
```

Comment:
We should probably find a better way to handle such cases of errors, but fine for now.